backend/tests/integration/test_b057_p3_webhook_ingestion_unblocking.py

```
@@ -313,3 +313,82 @@ def test_b057_p3_webhook_e2e_persists_under_runtime_identity(
+        "data": {
+            "object": {
+                "id": "pi_b057_p3_pii",
+                "amount": 4321,
+                "currency": "usd",
+                "receipt_email": "receipt@test.invalid",
+                "billing_details": {"email": "bill@test.invalid"},
+            }
+        },
+    }
+    pii_body = json.dumps(pii_payload, separators=(",", ":")).encode("utf-8")
+    pii_signed_payload = f"{created}.{pii_body.decode('utf-8')}".encode("utf-8")
```

Ruff (0.15.0)

[warning] Unnecessary UTF-8 `encoding` argument to `encode`

Remove unnecessary `encoding` argument

(UP012)